src/ResourceManager/AnalysisServices/Commands.AnalysisServices/Commands/SetAzureRmAnalysisServicesServer.cs

```diff
@@ -63,10 +63,18 @@ public class SetAzureAnalysisServicesServer : AnalysisServicesCmdletBase
         [ValidateNotNullOrEmpty]
         public string BackupBlobContainerUri { get; set; }
 
+        [Parameter(ValueFromPipelineByPropertyName = true, Mandatory = false,
+            HelpMessage = "The gateway name for configuring on the server.")]
+        [ValidateNotNullOrEmpty]
+        public string GatewayName { get; set; }
+
         [Parameter(Mandatory = false)]
         public SwitchParameter PassThru { get; set; }
 
-        [Parameter(ValueFromPipelineByPropertyName = true, Position = 5, Mandatory = true,
+		[Parameter(Mandatory = false)]
```

Comment:
this was not part of the cmdlet review.  Dissociate is probably not the correct verb, you should like use Disconnect, or allow setting the GatewayName to 'ull or empty

Reply:
Will do.

Reply:
Fixed.